### DISCUSSION TITLE
EdgrtX 2.8 instrall but cannot prep SD card. Get message  saying SD Card needs Conversion ?

### DISCUSSION THREAD

**Discussion by Airbrushkid19** [EdgeTX/edgetx#3012] [General]
I have been for months try to switch from OpenTX to EdgeTX. And it all fails. I tried this one youtuber RC video reviews. Go  to do SD card prep. I will only select all languages. I pick English and picks all of them. I am only allowed to pick default usb device. When I click to go. I get a message saying DFU has to be turn off. I it OFF.

I try EdgeTX Buddy. Doesn't work. Tried 5 different computers with different OS's. It sucks that It only works on Chrome. It doesn't work on chrome on 5 different computers. Why? OS's win 7,iOS, Win 10. 

Can someone tell me why or how to fix.. Is there new release of EdgeTX flasher? was told on discord the flasher is broken.

**Comment by pfeerick** [comment]
Possibly some confusion as to what is needed.

For a firmware update, you need the radio in DFU mode (for most radios,
this is done by connecting the radio to the computer with the radio turned
off).

To update/prepare the SD Card, the radio needs to be on, and in bootloader
mode (or you can use a SD Card reader) ... hence the mention that DFU needs
to be off. In other words, you should be able to see the radios SD Card in
File Explorer when you want to update/prepare it.

**Reply by Airbrushkid19** [reply]
Thank you for stepping in. I forgot to mention that the 2.8 firmware was already install by me. After that everything when craps. I can go to bootloader mode. You see the 2.08 firmware is installed. I start the radio and I get Bad Radio Data. Now i did everything this guy said to do!!

https://www.youtube.com/watch?v=PhpDf-j_3_Q&list=LL&index=4

**Reply by Airbrushkid19** [reply]
I think I got it. I kept getting Bad Radio Data. All of a sudden I decide to just click all the buttons on the transmitter. The screen showed up. And kept getting a beeping sound. So I connected to the pc and the flasher software. Then to the SD card prep. Tried to prep the sd card.nothing. Pulled the sd card format on pc. Then got the sd card edgetx content from github  Copy it to the sd card put back in radio and power up radio it seems to be working. Why, I don't even know? Now I got to figure out how to set this radio up. Futaba and Spektrum  and JR radios are easy as hell this. This is messed up.

**Reply by pfeerick** [reply]
When you got the "Bad Radio Data" prompt, did you try pressing ENT (the roller) at all to dismiss the message? As this message is expected if you are using a blank SD card. And possibly also when comming from a significantly older version (i.e. OpenTX) where conversion of the radio settings format is required. By copying the SD card content from GitHub, you also created the RADIO folder, which should have instead made it skip to the calibration screen IIRC. OTX/ETX is a learning curve for sure. So was Spektrum when I started with that... and frankly... I will never go back to Spektrum as it's more confusing now that I've realised what is possible when the system isn't holding your hand and limiting what you can do all the time.

**Comment by Airbrushkid19** [comment]
Why would I have clicked the roller? I'm new to this OpenTX-EdgeTX .That video I watch about 44 times. The video was from RC Video Reviews YouTube Channel. He did not mention anything could go wrong! I clicked every button because I didn't know what to do!  There was a SD Card that was with the radio. The radio is new and had OpenTX on it when I bought it New. I was going to try to set up a heli but. Don't understand how to. The menus don't list things, like Throttle Curve, Pitch Curve, Throttle Hold, Throttle Cut. Or even Idle up 1, 2 and 3! You cannot even figure out how to Calibrate the Sticks. Or how to make a model like airplane, quad or helicopter.

**Reply by Airbrushkid19** [reply]
I am happy to say that I started to figure it out. I only plan on using this TX16S for FPV Quads and sim on my computer. I've been flying heli's for over 55 years and always use Futaba from the old days. When they didn't have any of this touch screen, program stuff right up to my Futaba 16iz. Never in 55 years has a Futaba radio failed or cause any problems. I will stay with Futaba for my heli's and planes. I trust Futaba radios more then any others out there. I will leave it at that. 

So thank you
